### DISCUSSION TITLE
SSL not working

### DISCUSSION THREAD

**Discussion by Alinaswe3** [vercel/community#2813] [Help] [answered]
### Summary

Guys I recently deployed my website to Vercel hosting, and I read that Vercel will automatically provide an SSL certificate for me. When I visit the site, it tells me it is not secure. How do I resolve that? 

### Example

www.alinaswecodes.com

### Steps to Reproduce

Just visit the site and see if SSL is working.

**Comment by swarnava** [comment, marked as answer]
Hi,

It looks like DNS is propagated successfully and SSL is installed now. Are you still facing issue?

**Comment by Alinaswe3** [comment]
No am not, thank you very much for your timely response. Sent from Mail for Windows From: Swarnava SenguptaSent: Saturday, June 10, 2023 12:11 PMTo: vercel/communityCc: Alinaswe; AuthorSubject: Re: [vercel/community] SSL not working (Discussion #2813) Hi,It looks like DNS is propagated successfully and SSL is installed now. Are you still facing issue?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
